src/userdb/userdbctl.c

```diff
@@ -1130,6 +1140,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "  -I                         Equivalent to --disposition=intrinsic\n"
                "  -S                         Equivalent to --disposition=system\n"
                "  -R                         Equivalent to --disposition=regular\n"
+               "  -B --boundaries=no         Hide UID/GID range boundaries in output\n"
```

Comment:
I'd like to split the lines into two, `--boundaries=BOOL` and `-B`, even though `--boundaries=yes` is mostly useless.